packages/wire-format/src/types.ts

```diff
@@ -146,6 +146,7 @@ type GetChannel = {type: 'GetChannel'; channelId: Bytes32};
 export type ChannelRequest = GetChannel;
 
 export interface Payload {
+  walletVersion: string; // e.g. @statechannels/server-wallet@1.4.0
```

Comment:
New required property. This is a breaking change (I did not mark it as such in my commit, to avoid bumping  major version number).